### PR TITLE
Bump MinI, MySQL and RabbitMQ helm dependencies to latest version

### PR DESCRIPTION
**Pull request by csweichel**
## Description
We were using a very outdated MinIO, MySQL and RabbitMQ helm chart version. These version have become unavailable since. This PR bumps the helm charts to their latest version.

image

## How to test
```bash
cd install/installer/third_party/charts/minio
helm dep update
cd ../mysql
helm dep update
cd ../rabbitmq
helm dep update
```

## Release Notes

```release-note
[installer] Bump MinIO helm chart to 11.6.3
[installer] Bump MySQL helm chart to 9.1.2
[installer] Bump RabbitMQ helm chart to 10.1.1
```

### CODE REVIEW

**Comment by MrSimonEmms** [comment]
Do we need to test this with a self-hosted in-cluster instance or have you done that?

I suspect I'll need to do a new hotfix release for the Installer/KOTS as a result of this. I'll do that job

**Review by MrSimonEmms — APPROVED** [review]
/hold holding in case of new changes, but lgtm

**Comment by csweichel** [comment]
> Do we need to test this with a self-hosted in-cluster instance or have you done that?

Yes, we do

> I suspect I'll need to do a new hotfix release for the Installer/KOTS as a result of this. I'll do that job

I don't think so - we're shipping the helm charts embedded in the installer.
This just breaks our builds.

**Comment by MrSimonEmms** [comment]
Ok, I'll do the test in the self-hosted instance now and approve when it's good

**Review by MrSimonEmms — APPROVED** [review]
Tested on my self-hosted instance and works fine - did a full image build in a fresh instance

**Comment by MrSimonEmms** [comment]
Ooops, didn't realise it was still on hold 😊

**Pull request merged**
